app/ui/src/main/java/com/fsck/k9/ui/helper/MaterialColors.java

```diff
@@ -0,0 +1,37 @@
+package com.fsck.k9.ui.helper;
+
+public class MaterialColors {
+    /*
+     * Heavily modified version of https://gist.github.com/kalehv/bae765c756e94455ed88
+     */
+    public static final int IDX_50  = 0;
+    public static final int IDX_100 = 1;
+    public static final int IDX_200 = 2;
+    public static final int IDX_300 = 3;
+    public static final int IDX_400 = 4;
+    public static final int IDX_500 = 5;
+    public static final int IDX_600 = 6;
+    public static final int IDX_700 = 7;
+    public static final int IDX_800 = 8;
+    public static final int IDX_900 = 9;
+
+    public static final int[] RED = {0xFFFFEBEE, 0xFFFFCDD2, 0xFFEF9A9A, 0xFFE57373, 0xFFEF5350, 0xFFF44336, 0xFFE53935, 0xFFD32F2F, 0xFFC62828, 0xFFB71C1C};
```

Comment:
Instead of using arrays I'd like us to use constants for all colors, i.e. `RED_50`, `RED_100`, `RED_200` etc.

I think it will make this class easier to understand, and it should make `ContactLetterBitmapCreator.BACKGROUND_COLORS_*` easier to read. It should also allow Proguard to get rid of `MaterialColors` entirely.